src/fixit/config.py

```diff
@@ -23,6 +40,12 @@
 log = logging.getLogger(__name__)
 
 
+class ConfigError(ValueError):
+    def __init__(self, msg: str, config: RawConfig):
```

Comment:
Hmm, do we still get typechecking on this function without the return signature?
```suggestion
    def __init__(self, msg: str, config: RawConfig) -> None:
```

Reply:
I'm not sure about Pyre, but mypy has specific support for eliding the return type of `__init__` methods because they will never be anything else. See mypy#5677